doc/server/akkahttp.md

```diff
@@ -64,10 +69,24 @@ using tapir endpoint descriptions; or, that the tapir-generated route is wrapped
 "edge-case endpoints", which require some special logic not expressible using tapir, can be always implemented directly 
 using akka-http. For example:
 
-```scala
+```scala mdoc:invisible
+import sttp.tapir._
+import sttp.tapir.server.akkahttp._
+import akka.http.scaladsl.server._
+
+def metricsDirective: Directive0 = Directive { inner => ctx =>
```

Comment:
this can be visible, but we can leave the directives unimplemented just leaving the type signatures, e.g. `def metricsDirective: Directive0 = ???`. This describes the intention well without the need to invent empty implementations

Reply:
Good point... I thought I did it this way before and something didn't work. I did a double-take and now it works 👍